src/learning_gem5/part3/Kconfig

```diff
@@ -23,7 +23,10 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-Import('*')
+config PROTOCOL
+    default "MSI" if RubyProtocolMsi
 
-sticky_vars.Add(BoolVariable('BUILD_GPU', 'Build the compute-GPU model',
-                             False))
+cont_choice "Ruby protocol"
+    config RubyProtocolMsi
```

Comment:
Why are these `CamelCase` when all other options are all caps `SNAKE_CASE`?

What is the style rule here?

Reply:
Changed to SNAKE_CASE fomat